### PR TITLE
Fold in comments from is_callable() page

### PR DESCRIPTION
**Pull request by Crell**
I'm not sure this is structured properly.  I can restructure to some other format if someone tells me what the format should be. :-)

### CODE REVIEW

**Comment by morrisonlevi** [comment]
Maybe add a note on visibility of the `__invoke` method?

File: reference/var/functions/is-callable.xml [hunk 1]
```diff
@@ -69,6 +69,20 @@
   </para>
  </refsect1>
 
+ <refsect1>
+  <para>An object is always callable if it implements <function>__invoke()</function>.</para>
+  <para>A class name is callable if it implements <function>__callStatic()</function></para>
+  <para>
+   If an object implements <function>__call()</function>, then this function will
+   return true for any method on that object, even if the method is not defined.
+  </para>
```
Comment (cmb69) [inline reference/var/functions/is-callable.xml:78]: I think this can go into a single para.

File: reference/var/functions/is-callable.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
  <refnamediv>
   <refname>is_callable</refname>
   <refpurpose>
-   Verify that the contents of a variable can be called as a function
+   Verify that the contents of a variable can be called as a function from the current scope.
```
Comment (cmb69) [inline reference/var/functions/is-callable.xml:7]:
```suggestion
   Verify that a value can be called as a function from the current scope.
```

File: reference/var/functions/is-callable.xml [hunk 3]
```diff
@@ -69,6 +69,20 @@
   </para>
  </refsect1>
 
+ <refsect1>
```
Comment (cmb69) [inline reference/var/functions/is-callable.xml:72]: The refsect1 should have a role. If role="note", it should be put below the examples. But maybe the info is better suited in the description above?

**Review by cmb69 — COMMENTED** [review]
Thank you!

**Comment by Crell** [comment]
Revised.

File: reference/var/functions/is-callable.xml [hunk 4]
```diff
@@ -151,6 +151,23 @@ bool(false)
   </para>   
  </refsect1>
 
+ <refsect1 type="notes">
+  <simplelist>
+   <member><para>
+    An object is always callable if it implements <function>__invoke()</function>,
+    and that method is visible in the current scope.
+   </para></member>
```
Comment (cmb69) [inline reference/var/functions/is-callable.xml:159]: `<para>` is not allowed as child of `<member>`, so:
```suggestion
   <member>
    An object is always callable if it implements <function>__invoke()</function>,
    and that method is visible in the current scope.
   </member>
```

Comment (Crell) [reply reference/var/functions/is-callable.xml:159]: I thought it was the other way around, you had to include something inside member?

Comment (cmb69) [reply reference/var/functions/is-callable.xml]: You may confuse that with `<listitem>` which indeed needs a `<para>` or so.

File: reference/var/functions/is-callable.xml [hunk 5]
```diff
@@ -151,6 +151,23 @@ bool(false)
   </para>   
  </refsect1>
 
+ <refsect1 type="notes">
```
Comment (cmb69) [inline reference/var/functions/is-callable.xml:154]:
```suggestion
 <refsect1 role="notes">
```

**Comment by cmb69** [comment]
@girgias, what do you think about the layout of the notes section? It's rather different from what we usually use (i.e. multiple `<note>`s and such), but I think I like this better.

**Comment by Girgias** [comment]
The layout might make more sense then what we currently have at times, so it's all green on my end. :-)

File: reference/var/functions/is-callable.xml [hunk 6]
```diff
@@ -151,6 +151,23 @@ bool(false)
   </para>   
  </refsect1>
 
+ <refsect1 role="notes">
+  <simplelist>
+   <member>
+    An object is always callable if it implements <function>__invoke()</function>,
+    and that method is visible in the current scope.
+   </member>
+   <member>A class name is callable if it implements <function>__callStatic()</function></member>
```
Comment (salathe) [inline reference/var/functions/is-callable.xml:160]: `<function>__callStatic()</function>` should be `<link linkend="object.callstatic">__callStatic()</link>`

File: reference/var/functions/is-callable.xml [hunk 7]
```diff
@@ -151,6 +151,23 @@ bool(false)
...
+   <member>
+    If an object implements <function>__call()</function>, then this function will
```
Comment (salathe) [inline reference/var/functions/is-callable.xml:162]: `<function>__call()</function>` should be `<link linkend="object.call">__call()</link>`

File: reference/var/functions/is-callable.xml [hunk 8]
```diff
@@ -151,6 +151,23 @@ bool(false)
...
+    return true for any method on that object, even if the method is not defined.
```
Comment (salathe) [inline reference/var/functions/is-callable.xml:163]: `true` should be `&true;`

**Review by salathe — COMMENTED** [review]
Some post-merge comments. Poor effort from the code reviewers on this one. 😛

**Comment by cmb69** [comment]
Thank you, @salathe! Fixed as 83a17a7324c2597bd6385148abf19f76127229f5.

**Pull request merged**